### PR TITLE
Fix group comments and apostrophes

### PR DESCRIPTION
**Pull request by flodolo**
Not exactly sure if this is enough, hopefully tests are not looking for specific text.

CC @clouserw 

I tried running a linter on source files, and a few issues:
* Group comment should have a space before and after, and be followed by at least a string. I move the one about "email content" in the following file to avoid that problem.
* Strings should use `’` and not `'`.

### CODE REVIEW

**Comment by flodolo** [comment]
Also #12773 was found running that linter.

**Comment by clouserw** [comment]
Thanks @flodolo .  This looks good to me.  Is your linter able to be integrated into PR checks as part of github actions maybe?

**Review by clouserw — APPROVED** [review]

**Comment by flodolo** [comment]
> Thanks @flodolo . This looks good to me. Is your linter able to be integrated into PR checks as part of github actions maybe?

Yes, that would be the idea. 

It's still a bit rough on the edge (only a few hours old), but I'm starting to set it up on a few smaller projects.

File: packages/fxa-auth-server/lib/senders/emails/layouts/fxa/en.ftl [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+## Email content
```
Comment (LZoog) [inline packages/fxa-auth-server/lib/senders/emails/layouts/fxa/en.ftl:2]: Just a heads up that this is not guaranteed to be the first file concatenated to `auth.ftl`. If we notice in the future due to a new file that this is no longer grouping correctly we might want to move it back to `auth.ftl`.

**Pull request merged**